roadmap.html

```diff
@@ -36,9 +36,13 @@ <h3><a href="https://github.com/neovim/neovim/milestone/19">0.5</a></h3>
   <li>Extended marks (text properties, decorations, virtual text)</li>
 </ul>
 
-<h3>0.6</h3>
+<h3><a href="https://github.com/neovim/neovim/milestone/11">0.6</a></h3>
 <ul>
   <li>Better file-change detection</li>
+  <li>Terminal UI as a remote UI</li>
+  <li>Virtual lines (think virtualtext but on new lines)</li>
+  <li>Improved folds</li>
+  <li>Diagnostic API</li>
```

Comment:
assuming this is for LSP, or is a general UI subsystem enhancement?

```suggestion
  <li>LSP: Diagnostic API</li>
```

Reply:
I think it is meant to be generic, i e reuse the LSP UI logic even for non-LSP sources

Reply:
Yes, the motivation was for people to (be able to) stop shoehorning random linters into LSP via null-ls or efm while still getting decent-looking diagnostics (that don't fight each other).